### PR TITLE
Add test script for comment count

### PR DESCRIPTION
**Pull request by dev-mamun**
In This PR

- test comments counter with unit tests using Jest

### CODE REVIEW

File: src/commentCount.test.js
Comment (anita00001) [inline src/commentCount.test.js:1]: Kindly, place the file src/commentCount.test.js in the src/__tests__ directory in order to organize the files properly.

**Review by anita00001 — CHANGES_REQUESTED** [review]
Kindly, place the file src/commentCount.test.js in the src/tests directory in order to organize the files properly.

File: src/commentCount.test.js
Comment (dev-mamun) [reply src/commentCount.test.js:1]: Changed test file location.

**Review by anita00001 — APPROVED** [review]
Hi @dev-mamun , 

Your project is complete!  There is nothing else to say other than... it's time to merge it ⌛⌛
Congratulations! 🎉 



## Highlights

✅ No linter error
✅ Great test cases 👍

**Pull request merged**